SonarQube.Bootstrapper/BootstrapperSettings.cs

```diff
@@ -181,7 +181,7 @@ public LoggerVerbosity LoggingVerbosity
 
         private static string CalculateTempDir(ILogger logger)
         {
-            logger.LogDebug(Resources.INFO_UsingEnvVarToGetDirectory);
```

Comment:
I changed the prefix of all of the message resources from INFO_ or DEBUG_ to "MSG_" to avoid cases like this where we have LogDebug being called with what claims to be an info message.